README.md

```diff
@@ -125,3 +125,20 @@ Configuring Shibboleth
 * visit '/Shibboleth.sso/Metadata' to download and review the metadata, rename to a filename of your preference
 * upload the SP metadata to http://testshib.org/register.html or your Identiy Provider (IdP)
 * login by clicking on 'Sign into Shibboleth' and choosing one of the available identities
+
+Audit Fix
+---
+* To fix data with triple files:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'triples','sample-sparql.txt'```
+* To remove irrelevant theses dates:
+```RAILS_ENV = {{RAILS_ENV}} bundle exec ruby bin/fix/fix-audit.rb 'thesis'```
```

Comment:
@weiweishi even with the object you gave me this invocation (for both 'thesis' and 'reindex') seems to be expecting a second argument (see input = ARGV[1] on line 170. What should I try?